modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -25,6 +25,10 @@ def index
       def show
         super
       end
+
+      def fetch_aud
+        Settings.oidc.isolated_audience.claims
```

Comment:
So claims controller has nothing to do with the forms controllers so you may want to consider adding this to the application controller and just use the v0? flag to avoid using it in non Oauth scenarios. Also what calls the fetch_aud method?

Reply:
- I assumed from the ticket description that this was a change only intended for specifically the `/claims` endpoints. @dbrown-va is that an accurate reading, or should this actually go in the higher-level `claims_api` `application_controller`?
- `fetch_aud` gets called in the [`openid_application_controller`](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/openid_application_controller.rb#L47)

Reply:
Update: moved it up to the `application_controller`. Non-OAuth endpoints will skip the `:validate` call that utilizes this anyway

Reply:
This only applies to the OAuth endpoints for /claims and would otherwise be unused for non OAuth endpoints.

My understanding is there shouldn't be harm for it being in the application controller, but definitely not required at that top level.  You just need to ensure it overrides the base `fetch_aud` method in the `openid_application_controller` for any OAuth protected endpoints.